spec/index.bs

```diff
@@ -13599,12 +13599,22 @@ enum GPUCanvasAlphaMode {
     "premultiplied",
 };
 
+enum GPUCanvasColorMode {
+    "default",
+    "extended",
+};
+
+dictionary GPUCanvasColorMetadata {
+  GPUCanvasColorMode mode = "default";
+};
+
 dictionary GPUCanvasConfiguration {
     required GPUDevice device;
     required GPUTextureFormat format;
     GPUTextureUsageFlags usage = 0x10;  // GPUTextureUsage.RENDER_ATTACHMENT
     sequence<GPUTextureFormat> viewFormats = [];
     PredefinedColorSpace colorSpace = "srgb";
+    GPUCanvasColorMetadata colorMetadata;
```

Comment:
```suggestion
    GPUCanvasColorMetadata colorMetadata = {};
```

Reply:
Unless there should be some difference between not specifying colorMetadata and specifying the default colorMetadata.